pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -1544,11 +1549,17 @@ func Test_isSchedulableAfterNodeChange(t *testing.T) {
 			}
 			actualHint, err := p.(*Fit).isSchedulableAfterNodeChange(logger, tc.pod, tc.oldObj, tc.newObj)
 			if tc.expectedErr {
-				require.Error(t, err)
+				if err == nil {
+					t.Errorf("Expected error but got nil in isSchedulableAfterNodeChange")
+				}
 				return
 			}
-			require.NoError(t, err)
-			require.Equal(t, tc.expectedHint, actualHint)
+			if err != nil {
+				t.Errorf("Unexpected error in isSchedulableAfterNodeChange: %s", err.Error())
```

Comment:
You can skip `.Error()` here. Please change it in other places as well:
```suggestion
				t.Errorf("Unexpected error in isSchedulableAfterNodeChange: %s", err)
```